src/Currencies.java

```diff
@@ -0,0 +1,22 @@
+import java.util.*;
+public class Currencies {
+    Map<String, Map<String,Double>> rates;
```

Comment:
make this map private and fix formatting in the file. There is a shortcut for formatting whole file in intelij ( Ctrl+Alt+Shift+L)